lua/lvim/core/cmp.lua

```diff
@@ -224,7 +224,9 @@ M.config = function()
     },
     snippet = {
       expand = function(args)
-        require("luasnip").lsp_expand(args.body)
+        pcall(function()
+          require("luasnip").lsp_expand(args.body)
+        end)
```

Comment:
This shouldn't be necessary as this should only be called when a user expands a snippet. This shouldn't be called in headless mode since there is no input.